### DISCUSSION TITLE
Italian traslation

### DISCUSSION THREAD

**Discussion by ithinkmax** [Yue-Yang/ChatGPT-Siri#75] [General]
Its possible to translate the shortcut in Italian? If i try to use it on my device i get errors on all commands:

<a href="https://ibb.co/jJPGp4F"><img src="https://i.ibb.co/z6Tbkfw/IMG-2-EDADEA48-A13-1.jpg" alt="IMG-2-EDADEA48-A13-1" border="0" /></a>

Seems that key fuctions need to be translated...

**Comment by Yue-Yang** [comment]
You may need to add a payment method as there are fees for using the API after the end of your free trial.

https://github.com/Yue-Yang/ChatGPT-Siri/discussions/30

**Reply by ithinkmax** [reply]
Good morning Yue-Yang and thank you for your support and this wonderful shortcut.

I checked and my paid version is still active and from OpenAI site I am able to use chatgpt v4 plus without problem. But the problem still remains!
